<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN'
  'http://www.w3.org/TR/html4/strict.dtd'>
<html>
<head>
<title>Bio Links</title>
<link rel='icon' href='/favicon.png' type='image/png'/>
<link rel='stylesheet' href='links.css'>
</head>
<body>
<h1>Bio Links</h1>
<div><h2>UCSC</h2><ul>
<li><a href='http://genome.ucsc.edu'>Genome Home</a></li>
<li><a href='http://genome.ucsc.edu/cgi-bin/hgTracks'>Genome Browser</a></li>
<li><a href='http://genome.ucsc.edu/FAQ/FAQformat'>File Formats</a></li>
</ul></div>

<div><h2>NCBI</h2><ul>
<li><a href='http://www.ncbi.nlm.nih.gov'>Home</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/nuccore'>NCBI Nucleotide</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/refseq'>RefSeq</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/genomes/leuks.cgi'>Genome Index</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/genomes/genlist.cgi?taxid=2759&type=8&name=Eukaryotae%20Complete%20Chromosomes'>Genome Stats</a></li>
<li><a href='http://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html'>GenBank</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?DB=pubmed'>PubMed</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/nuccore/NR_003287'>The ubiquitous 28S</a></li>
<li><a href='ftp://ftp.ncbi.nih.gov/refseq'>RefSeq FTP</a></li>
<li><a href='ftp://ftp.ncbi.nih.gov/genomes'>Genome FTP</a></li>
</ul></div>

<div><h2>Tools</h2><ul>
<li><a href='http://seqanswers.com/'>SeqAnswers</a></li>
<li><a href='http://biopython.org/DIST/docs/tutorial/Tutorial.html'>BioPython</a></li>
<li><a href='http://www.bioconductor.org/'>BioConductor</a></li>
<li><a href='http://helix.nih.gov/docs/gcg'>GCG</a></li>
<li><a href='http://helixweb.nih.gov'>HelixWeb</a></li>
<li><a href='http://www-genome.wi.mit.edu/cgi-bin/primer/primer3.cgi/primer3_www.cgi'>Primer3</a></li>
<li><a href='http://www.ebi.ac.uk/embl/Documentation/User_manual/usrman.html'>EMBL</a></li>
</ul></div>

<p>
<div><h2>BWA</h2><ul>
<li><a href='http://bio-bwa.sourceforge.net'>BWA</a></li>
<li><a href='http://bio-bwa.sourceforge.net/bwa.shtml'>bwa(1)</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234/?tool=pmcentrez'>BWA - Fast & accurate...</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/pubmed/20080505'>BWA - Fast & accurate...</a></li>
</ul></div>

<div><h2>Bowtie</h2><ul>
<li><a href='http://bowtie-bio.sourceforge.net/index.shtml'>Bowtie</a></li>
<li><a href='http://bowtie-bio.sourceforge.net/manual.shtml'>Bowtie Manual</a></li>
<li><a href='http://cbcb.umd.edu/~langmead/NCBI_Nov2008.pdf'>Bowtie Pres PDF</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690996/?tool=pmcentrez'>Ultrafast & mem eff</a></li>
<li><a href='http://cufflinks.cbcb.umd.edu/'>Cufflinks</a></li>
<li><a href='http://tophat.cbcb.umd.edu/'>Tophat</a></li>
<li><a href='http://bowtie-bio.sourceforge.net/myrna/index.shtml'>Myrna</a></li>
</ul></div>

<div><h2>Broad Institute</h2><ul>
<li><a href='http://www.broad.mit.edu'>Broad Institute</a></li>
<li><a href='http://picard.sourceforge.net/'>Picard</a></li>
<li><a href='http://sourceforge.net/projects/picard/'>Picard Project</a></li>
<li><a href='http://samtools.sourceforge.net/'>SAMTools</a></li>
<li><a href='http://samtools.sourceforge.net/samtools.shtml'>SAMTools(1)</a></li>
<li><a href='http://www.sdforum.org/index.cfm?fuseaction=Calendar.eventDetail&eventID=13670'>SAM Sig</a></li>
<li><a href='http://www.broadinstitute.org/gsa/wiki/index.php/Main_Page'>GSA</a></li>
<li><a href='http://www.broadinstitute.org/gsa/wiki/index.php/The_Genome_Analysis_Toolkit'>GATK</a></li>
<li><a href='http://www.broadinstitute.org/gsa/wiki/index.php/The_Genome_Analysis_Toolkit#GATK_Tools_in_1.0_Release_Version'>GATK Tools</a></li>
<li><a href='http://www.broadinstitute.org/gsa/wiki/index.php/Unified_genotyper'>Unified genotyper</a></li>
<li><a href='http://www.broadinstitute.org/gsa/wiki/index.php/VariantEval'>VariantEval</a></li>
</ul></div>

<p>
<div><h2>EMBOSS</h2><ul>
<li><a href='http://emboss.sourceforge.net'>EMBOSS sf</a></li>
<li><a href='http://emboss.sourceforge.net/apps/release/6.2/emboss/apps/'>Apps Index</a></li>
<li><a href='http://emboss.sourceforge.net/apps/release/6.2/emboss/apps/infoseq.html'>infoseq</a></li>
<li><a href='http://emboss.sourceforge.net/apps/release/6.2/emboss/apps/seqret.html'>seqret</a></li>
<li><a href='http://helix.nih.gov/Applications/emboss-gcg.html'>GCG Equivs</a></li>
<li><a href='http://bioweb.pasteur.fr/nucleic/intro-en.html'>Tools index</a></li>
<li><a href='http://emboss.sourceforge.net/docs/emboss_tutorial/emboss_tutorial.html'>Tutorial</a></li>
<li><a href='http://emboss.sourceforge.net/docs/adminguide/admin.html'>Admin Guide</a></li>
</ul></div>

<div><h2>Staden</h2><ul>
<li><a href='http://staden.sourceforge.net'>Staden Package</a></li>
<li><a href='http://groups.google.com/group/bionet.software.staden'>Google Group</a></li>
</ul></div>

<div><h2>Galaxy</h2><ul>
<li><a href='http://main.g2.bx.psu.edu/'>Galaxy Home </a></li>
<li><a href='http://bitbucket.org/galaxy/galaxy-central/wiki/Home'>Galaxy Wiki </a></li>
<li><a href='http://bitbucket.org/galaxy/galaxy-central/wiki/LIMS/nglims'>Galaxy nglims</a></li>
<li><a href='http://bitbucket.org/galaxy/galaxy-central/wiki/Config/ApacheProxy'>Proxy Conf</a></li>
</ul></div>

<p>
<div><h2>1000 Genomes</h2><ul>
<li><a href='http://1000genomes.org/wiki/doku.php?id=1000_genomes:analysis:variant_call_format'>Variant Call Format</a></li>
<li><a href='http://1000genomes.org/wiki/doku.php?id=1000_genomes:analysis:vcf4.0'>Variant Call Format 4.0</a></li>
</ul></div>

<div><h2>Alignment Software</h2><ul>
<li><a href='http://www.irisa.fr/symbiose/projects/gassst/'>GASSST</a></li>
<li><a href='http://en.wikipedia.org/wiki/List_of_sequence_alignment_software'>Alignment Software</a></li>
<li><a href='http://seqanswers.com/forums/showthread.php?t=43'>SeqAnswers SW List</a></li>
<li><a href='http://soap.genomics.org.cn/'>SOAP</a></li>
<li><a href='http://maq.sourceforge.net/'>MAQ</a></li>
<li><a href='https://dnanexus.com/'>DNA Nexus</a></li>
</ul></div>

<div><h2>Workflow</h2><ul>
<li><a href='http://bioextract.org/GuestLogin'>BioExtract</a></li>
</ul></div>

<p>
<div><h2>UnaFold</h2><ul>
<li><a href='http://www.bioinfo.rpi.edu/applications/unafold/Software/'>DINAMelt Server</a></li>
<li><a href='http://www.bioinfo.rpi.edu/applications/unafold/Software/hybrid-ss-min.php'>hybrid-ss-min</a></li>
</ul></div>

<div><h2>WindowMasker</h2><ul>
<li><a href='http://www.ncbi.nlm.nih.gov/pubmed/16287941'>WindowMasker</a></li>
<li><a href='http://bioinformatics.oxfordjournals.org/cgi/content/full/22/2/134'>WindowMasker Paper</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/sites/entrez?db=toolkit'>C++ Toolkit Home</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/IEB/ToolBox/CPP_DOC/doxyhtml/modules.html'>C++ Toolkit Modules</a></li>
</ul></div>

<div><h2>Orgs</h2><ul>
<li><a href='http://www.baybifx.com'>BayBIFX </a></li>
<li><a href='http://etl.stanford.edu/'>Stanford ETL</a></li>
</ul></div>

<p>
<div><h2>Universities</h2><ul>
<li><a href='http://www.sanger.ac.uk/Software/formats/GFF/'>Sanger GFF</a></li>
<li><a href='http://genome-www.stanford.edu'>Stanford</a></li>
<li><a href='http://wi.mit.edu'>The Whitehead Inst.</a></li>
<li><a href='http://genome.wustl.edu/genome_group.cgi'>WUSTL</a></li>
</ul></div>

<div><h2>Centers</h2><ul>
<li><a href='http://www.sanger.ac.uk'>Sanger</a></li>
<li><a href='http://microrna.sanger.ac.uk/sequences/index.shtml'>Sanger miRBase</a></li>
<li><a href='http://ncicb.nci.nih.gov'>NCI</a></li>
<li><a href='https://wiki.nci.nih.gov/display/ICR/caBIO'>caBIO</a></li>
<li><a href='http://www.jcvi.org/'>J Craig Venter Inst</a></li>
<li><a href='http://www.genome.jp/'>GenomeNet</a></li>
<li><a href='http://www.genome.jp/kegg'>Kyoto Enc Genes Genomes</a></li>
<li><a href='http://www.genome.jp/kegg/catalog/org_list.html'>KEGG Genome Tree</a></li>
<li><a href='http://www.wormbase.org/'>WormBase </a></li>
</ul></div>

<div><h2>Tm Functions</h2><ul>
<li><a href='http://www.sigmaaldrich.com/Brands/Sigma_Genosys/Custom_DNA/Key_Resources/Oligos_Melting_Temp.html'>Sigma Aldrich</a></li>
<li><a href='http://www.pnas.org/cgi/content/full/95/4/1460'>PNAS</a></li>
<li><a href='http://genamics.com/expression/primer.htm'>Genamics</a></li>
<li><a href='http://bioinformatics.oxfordjournals.org/cgi/content/full/21/20/3918?ck=nck'>Tm Paper</a></li>
<li><a href='http://pubs.acs.org/cgi-bin/abstract.cgi/bichaw/1996/35/i11/abs/bi951907q.html'>Tm NN</a></li>
</ul></div>

<p>
<div><h2>More Tools</h2><ul>
<li><a href='http://www.gnu.org/software/gawk/manual/gawkinet/html_node/PROTBASE.html'>Gawk</a></li>
<li><a href='http://www.basic.northwestern.edu/biotools/oligocalc.html'>NW OligoCalc</a></li>
<li><a href='http://www.pnas.org/cgi/content/full/95/4/1460'>A unified view n-n </a></li>
<li><a href='http://www.mathworks.com/access/helpdesk/help/toolbox/bioinfo/?/access/helpdesk/help/toolbox/bioinfo/bioinfo_product_page.html&http://www.mathworks.com/access/helpdesk/help/toolbox/bioinfo/topbanner.html'>Bioinformatics Toolbox </a></li>
<li><a href='http://nar.oxfordjournals.org/cgi/content/full/25/20/4098'>CD </a></li>
<li><a href='http://www.clcbio.com/index.php?id=569'>Detailed Probe Parameters </a></li>
</ul></div>

<div><h2>Ensembl</h2><ul>
<li><a href='http://www.ensembl.org'>Home</a></li>
<li><a href='http://www.ensembl.org/Homo_sapiens/geneview'>Hs GeneView</a></li>
<li><a href='http://www.ensembl.org/Homo_sapiens/exonview'>Hs ExonView</a></li>
</ul></div>

<div><h2>Links</h2><ul>
<li><a href='http://ucsu.colorado.edu/~robeson'>Mac OS X Links</a></li>
<li><a href='http://www.ebioinformatics.org'>eBiotools</a></li>
<li><a href='http://www.dnaseq.co.uk/Chromatogram_viewers.html'>Chromatogram Viewers</a></li>
<li><a href='http://www.hgsc.bcm.tmc.edu/downloads/software/trace_viewer'>BCM TraceViewer</a></li>
<li><a href='http://www.littlest.co.uk/software/bioinf/old_packages/TraceView'>Trace Browser Paper</a></li>
</ul></div>

<p>
<div><h2>Mammal</h2><ul>
<li><a href='http://www.ncbi.nlm.nih.gov/genome/guide/human/'>H. sapiens</a></li>
<li><a href='http://www.ncbi.nlm.nih.gov/genome/guide/mouse/'>M. musculus</a></li>
<li><a href='http://www.hgsc.bcm.tmc.edu/projects/rmacaque/'>M. mulatta</a></li>
<li><a href='http://www.hgsc.bcm.tmc.edu/projects/bovine/'>B. taurus</a></li>
<li><a href='http://broadinstitute.org/mammals/dog'>C. familiarus</a></li>
<li><a href='http://broadinstitute.org/mammals/opossum'>M. domestica</a></li>
</ul></div>

<div><h2>Insect</h2><ul>
<li><a href='http://www.fruitfly.org/'>D. melanogaster</a></li>
<li><a href='http://www.hgsc.bcm.tmc.edu/projects/drosophila/pseudoobscura.html'>D. pseudoobscura</a></li>
<li><a href='http://www.hgsc.bcm.tmc.edu/projects/honeybee/'>A. mellifera</a></li>
<li><a href='http://agambiae.vectorbase.org/'>A. gambiae</a></li>
<li><a href='http://flybase.org/'>Flybase</a></li>
</ul></div>

<div><h2>Aquatic</h2><ul>
<li><a href='http://www.fugu-sg.org/'>T. ruripes</a></li>
<li><a href='http://www.sanger.ac.uk/Projects/D_rerio/'>D. rerio</a></li>
<li><a href='http://broadinstitute.org/annotation/tetraodon/'>T. nigroviridis</a></li>
<li><a href='http://mendel.stanford.edu/sidowlab/ciona.html'>C. savignyi</a></li>
<li><a href='http://genome.jgi-psf.org/Xentr4/Xentr4.home.html'>X. tropicalis</a></li>
<li><a href='http://genome.jgi-psf.org/Cioin2/Cioin2.home.html'>C. intestinalis (sea suirt)</a></li>
<li><a href='http://genome.jgi-psf.org/Chlre3/Chlre3.home.html'>C. reinhardtii (algae)</a></li>
</ul></div>

<p>
<div><h2>Plant</h2><ul>
<li><a href='http://www.tigr.org/plantProjects.shtml'>TIGR</a></li>
<li><a href='http://www.gramene.org/'>Grasses</a></li>
<li><a href='http://maize.tigr.org/'>Corn</a></li>
<li><a href='http://wheat.tigr.org/'>Wheat</a></li>
<li><a href='http://www.plantgdb.org/AtGDB/'>A. thaliana</a></li>
<li><a href='http://rice.plantbiology.msu.edu/'>O. sativa</a></li>
<li><a href='http://rgp.dna.affrc.go.jp/E/IRGSP/Build5/build5.html'>Rice genome</a></li>
<li><a href='ftp://ftp.plantbiology.msu.edu/pub/data/TIGR_Plant_Repeats/'>Plant repeats</a></li>
</ul></div>

<div><h2>SNPs</h2><ul>
<li><a href='http://www.ncbi.nlm.nih.gov/SNP'>dbSNP</a></li>
<li><a href='http://www.hapmap.org'>HapMap</a></li>
<li><a href='http://snp.cshl.org'>TSC</a></li>
</ul></div>

<div><h2>ABI</h2><ul>
<li><a href='http://www.appliedbiosystems.com'>Home</a></li>
<li><a href='https://products.appliedbiosystems.com/ab/en/US/adirect/ab;jsessionid=F3w4TVCLkYJMCMh9ptQybybz8DGdFB5j9t32mVJXThbKkmSJSGJv!-1307528326?cmd=catNavigate2&catID=600522'>DNA Sequencing</a></li>
</ul></div>

<p>
<div><h2>Publications</h2><ul>
<li><a href='http://www.americanscientist.org'>American Scientist</a></li>
<li><a href='http://www.sciam.com'>Scientific American</a></li>
<li><a href='http://www.scimag.com'>Scientific Computing</a></li>
<li><a href='http://www.genome-technology.com'>Genome Technology</a></li>
<li><a href='http://www.the-scientist.com'>The Scientist</a></li>
<li><a href='http://lib.bioinfo.pl/auth:Eberle,J'>Jim Eberle</a></li>
</ul></div>

<div><h2>Journals</h2><ul>
<li><a href='http://www.genome.org'>Genome Research</a></li>
<li><a href='http://www.nature.com/biotech'>Nature Biotechnology</a></li>
<li><a href='http://www.nature.com/nrg'>Nature Review Genetics</a></li>
<li><a href='http://www.sciencemag.org'>Science Magazine</a></li>
<li><a href='http://www.trends.com/tig/default.htm'>Trends in Genetics</a></li>
</ul></div>

<div><h2>Directories</h2><ul>
<li><a href='http://directory.google.com/Top/Science/Biology/Genetics'>Genetics</a></li>
<li><a href='http://directory.google.com/Top/Science/Biology/Bioinformatics/Online_Services'>Online Services</a></li>
<li><a href='http://www.vanderbilt.edu/bioinformatics/'>Vanderbilt Directory</a></li>
</ul></div>

</body>
</html>
